content/en/security_platform/cloud_siem/log_detection_rules.md

```diff
@@ -126,6 +134,37 @@ Anomaly detection inspects how the `group by` attribute has behaved in the past.
 
 **Note**: The query applies to all Datadog events and ingested logs that do not require indexing.
 
+{{% /tab %}}
+
+{{% tab "Impossible Travel" %}}
+
+### Search query
+
+Construct a search query using the same logic as a [log explorer search][1]. All logs matching this query are analyzed for a potential impossible travel. You can use the `preview` section to see which logs are matched by the current query.
+
+#### User attribute
+
+For the `user attribute`, select the field in the analyzed log that contains the user ID. This can be an identifier like an email address, user name, account identifier.
```

Comment:
```suggestion
For the `user attribute`, select the field in the analyzed log that contains the user ID. This can be an identifier like an email address, user name, or account identifier.
```